### PR TITLE
New files of Asma website

### PR DESCRIPTION
**Pull request by AsmaManaiSE**

### CODE REVIEW

File: content/about/employees/Asma/index.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+title: "Asma Manai"
+date: 2021-08-01
+resources:
+  - name: photo
+    src: Photo_asma.jpg
+linkedin: https://www.linkedin.com/in/asma-manai-a094b8102/
+email: asma@xal.no
+---
+Asma is an engineer and Ph.D in civil engineering. Her research topic is within the field of Structural Engineering. In particular, her focus is within the assessment and retrofitting of steel structures. This includes modeling, synthesis, characterization, and probabilistic analysis of steel structures. In more detail, her expertise is in analytical and numerical studies of pre-fatigued treated cracked structures. She makes use of these experiences for engineering applications in order to facilitate an extension of the service life of existing steel structures, such as bridges, offshore structures, and wind turbine.
```
Comment (mandus) [inline content/about/employees/Asma/index.md:10]: Maybe a bit nit-picking, but is it better to use past tense in the sentence `Her research topic is within...` as it reflects back to your prior work given that we aren't really a research organization?

Comment (AsmaManaiSE) [reply content/about/employees/Asma/index.md]: Done

File: content/about/employees/Asma/index.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+---
+title: "Asma Manai"
+date: 2021-08-01
+resources:
+  - name: photo
+    src: Photo_asma.jpg
+linkedin: https://www.linkedin.com/in/asma-manai-a094b8102/
+email: asma@xal.no
+---
+Asma is an engineer and Ph.D in civil engineering. Her research topic was within the field of Structural Engineering. In particular, her focus was within the assessment and retrofitting of steel structures. This included modeling, synthesis, characterization, and probabilistic analysis of steel structures. In more detail, her expertise was in analytical and numerical studies of pre-fatigued treated cracked structures. She made use of these experiences for engineering applications in order to facilitate an extension of the service life of existing steel structures, such as bridges, offshore structures, and wind turbine.
```
Comment (mandus) [inline content/about/employees/Asma/index.md:10]:
```suggestion
Asma is an engineer and Ph.D in civil engineering. Her research topic was within the field of Structural Engineering. In particular, her focus was within the assessment and retrofitting of steel structures. This included modeling, synthesis, characterization, and probabilistic analysis of steel structures. Her main expertise is thus in analytical and numerical studies of pre-fatigued treated cracked structures. In her role prior to joining Expert Analytics, Asma made use of these experiences for engineering applications in order to facilitate an extension of the service life of existing steel structures, such as bridges, offshore structures, and wind turbine.
```

Comment (mandus) [reply content/about/employees/Asma/index.md:10]: Tried to adjust a bit further to also tie in your previous work with the current role and your main expertice now. What do you think?

Comment (AsmaManaiSE) [reply content/about/employees/Asma/index.md:10]: sounds good!

**Review by mandus — APPROVED** [review]
looks good to me - let's keep it like this for now, and we can always update when you get some more xal-related projects :)

**Pull request merged**